### PR TITLE
1.19.4 Update

### PR DESCRIPTION
**Pull request by ChampionAsh5357**
Updates the documentation to use things from 1.19.4

### CODE REVIEW

File: docs/datagen/client/modelproviders.md [hunk 1]
```diff
@@ -393,7 +393,7 @@ Once the data provider is running, the models within the `ModelProvider` subclas
 @Override
 public CompletableFuture<?> run(CachedOutput cache) {
   // Populate the model provider
-  this.example.generateAll(cache); // Generate the models
+  CompletableFuture<?> exampleFutures = this.example.generateAll(cache); // Generate the models
```
Comment (sciwhiz12) [inline docs/datagen/client/modelproviders.md:396]: Should this show combining the given `CompletableFuture` with the one from its own `generateAll`? As in, showing the use of `CompletableFuture.allOf` (or another suitable alternative) wherein all CFs from calls to `generateAll`s are passed in as arguments?

File: docs/rendering/modelloaders/transform.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+When an [`BakedModel`][bakedmodel] is being rendered as an item, it can apply special handling depending on which transform it is being rendered in. "Transform" means in what context the model is being rendered. The possible transforms are represented in code by the `ItemDisplayContext` enum. There are two systems for handling transform: the deprecated vanilla system, constituted by `BakedModel#getTransforms`, `ItemTransforms`, and `ItemTransform`, and the Forge system, embodied by the method `IForgeBakedModel#applyTransform`. The vanilla code is patched to favor using `applyTransform` over the vanilla system whenever possible.
 
-`TransformType`
+`ItemDisplayContext`
 ---------------
 
-`NONE` - Unused.
+`NONE` - Used for the display entity and by Forge when a `Block`'s `RenderShape` is set to `#ENTITYBLOCK_ANIMATED`.
```
Comment (sciwhiz12) [inline docs/rendering/modelloaders/transform.md]: Should this note that it is used by the display entity by default when no other item display context is set?

**Review by sciwhiz12 — CHANGES_REQUESTED** [review]

File: docs/datagen/client/modelproviders.md (hunk 1 above)
Comment (ChampionAsh5357) [reply docs/datagen/client/modelproviders.md:396]: Sure, that's fine.

File: docs/rendering/modelloaders/transform.md (hunk 2 above)
Comment (ChampionAsh5357) [reply docs/rendering/modelloaders/transform.md]: Sure, the display entity is for debugging purposes.

**Review by sciwhiz12 — APPROVED** [review]

**Pull request merged**